site.yml

```diff
@@ -292,6 +292,22 @@
       tags: ['openstack', 'horizon', 'control']
   environment: "{{ env_vars|default({}) }}"
 
+- name: install swift-common
+  hosts: swiftnode
+  any_errors_fatal: true
+  roles:
+    - role: swift-common
+      tags: ['openstack', 'swift', 'swift-common', 'data']
+  environment: "{{ env_vars|default({}) }}"
+
+- name: swift bootstrap rings
```

Comment:
swift-ring role has dependency on swift-common.. so previous block may not be needed?